src/Core/Services/GraphQLSchemaCreator.cs

```diff
@@ -267,6 +282,21 @@ private DocumentNode GenerateCosmosGraphQLObjects(HashSet<string> dataSourceName
             return root;
         }
 
+        /// <summary>
+        /// Create and return a default GraphQL result field for a mutation which doesn't
+        /// define a result set and doesn't return any rows.
+        /// </summary>
+        private static FieldDefinitionNode GetDbOperationResultField()
+        {
+            return new(
+                location: null,
+                name: new("result"),
```

Comment:
define a constant string for this "result" field name, in GraphQLUtils perhaps?

Reply:
fixed.